src/Thumbnail/FileThumbnail.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Thumbnail;
+
+use Sonata\MediaBundle\Model\MediaInterface;
+use Sonata\MediaBundle\Provider\MediaProviderInterface;
+use Symfony\Component\Asset\Packages;
+
+/**
+ * @author Jordi Sala Morales <jordism91@gmail.com>
+ */
+final class FileThumbnail implements ThumbnailInterface
```

Comment:
The namespace already includes the term "Thumbnail".
IMO, "File" is enough for this class.

Reply:
I don't agree with this, or the interface could have been called `Interface` since it's in the Thumbnail namespace.
It avoids conflict with another File class and helps to give context when using the class.
The `ArrayCollection` is not called `Array` and there is lot of similar example.

Reply:
All the code on this bundle and other sonata bundle are the same way. On this same folder for example, all other Thumbnails are called that way. Or you can check for Providers, all include the folder name again.

This is for consistency, but also I don't agree calling it simply "File", it could be misleading, it is not a file.